gnomad/sample_qc/pipeline.py

```diff
@@ -331,13 +342,16 @@ def annotate_sex(
                     if variants_only_y_ploidy
                     else f"{normalization_contig}_mean_dp",
                 )
-                if variants_only_x_ploidy:
+                ploidy_ht = ploidy_ht.annotate(**y_ploidy_ht[ploidy_ht.key])
+                # If the `variants_only_y_ploidy' is True modify the name of the normalization contig mean DP to indicate
+                # that this is the variant dataset only mean DP (this will have already been added if
+                # 'variants_only_x_ploidy' was also True.
+                if variants_only_y_ploidy:
```

Comment:
i think this renaming was already taken care of in line 342

Reply:
great point, I think that above is actually wrong because `impute_sex_ploidy` will return it as `{normalization_contig}_mean_dp` so I think above needs to be changed to only select that and then here we rename it

Reply:
ah ok, and then should there also be a renaming for if variants_only_x_ploidy was true after line 326?

Reply:
Very right, I added the change